templates/files/pimcore/phpstan.neon

```diff
@@ -23,3 +22,10 @@ parameters:
   # Point PHPStan to internal constants that are bootstrapped
   bootstrapFiles:
     - ./vendor/youwe/testing-suite/config/pimcore/phpstan-bootstrap.php
+
+  ignoreErrors:
+    # PHPStan won't correctly recognize "@var Pimcore\Model\DataObject\ClassName $this" what we so often use in abstract
```

Comment:
'what' should be 'which'. Or rephrase to something like: , something we often use...